resources/lang/de/settings.php

```diff
@@ -7,6 +7,11 @@
     "connect" => "Verbinden",
     "connected" => "Verbunden",
     "current-password" => "Aktuelles Passwort",
+    "delete-profile-picture" => "Profilbild löschen",
+    "delete-profile-picture-btn" => "Profilbild löschen",
+    "delete-profile-picture-desc" => "Du möchtest dein Profilbild löschen. Diese Aktion kann <strong>nicht rückgängig</strong> gemacht werden. Bist du dir sicher?",
+    "delete-profile-picture-yes" => "Ja, weg damit.",
+    "delete-profile-picture-no" => "Nein, bring mich zurück!",
```

Comment:
Bisschen weniger umgangssprachlich, bitte

```suggestion
    "delete-profile-picture-yes" => "Wirklich löschen",
    "delete-profile-picture-no" => "Zurück",
```